go/minimal/main.go

```diff
@@ -27,18 +27,12 @@ func main() {
 		ClientID:     os.Getenv("ENSIGN_CLIENT_ID"),
 		ClientSecret: os.Getenv("ENSIGN_CLIENT_SECRET"),
 		// AuthURL:      "https://auth.ensign.world", // uncomment if you are in staging
-		// Endpoint:     "staging.ensign.world:443", // uncomment if you are in staging
+		// Endpoint:     "staging.ensign.world:443",  // uncomment if you are in staging
 	})
 	if err != nil {
 		panic(fmt.Errorf("could not create client: %s", err))
 	}
 
-	// Create Ensign Publisher
-	pub, err := client.Publish(context.Background())
-	if err != nil {
-		panic(fmt.Errorf("could not create publisher: %s", err))
-	}
```

Comment:
This is a known issue - if you connect a publisher or subscriber before the topic is created; then the publisher/subscriber stream won't know about the new stream, so moving this under the create topic snippet fixed the problem and the event was able to be published.